hierarchicalforecast/utils.py

```diff
@@ -68,7 +68,75 @@ def cov2corr(cov, return_std=False):
     else:
         return corr
 
+# %% ../nbs/utils.ipynb 7
+# convert levels to output quantile names
+def level_to_outputs(level):
```

Comment:
Let's add the following docstring:
"""
Converts list of levels into output names matching StatsForecast and NeuralForecast methods.

**Parameters:**<br>
`level`: int list [0,100]. Probability levels for prediction intervals.

**Returns:**<br>
`output_names`: str list. String list with output column names.
"""